packages/react-reconciler/src/ReactFiberReconciler.js

```diff
@@ -511,7 +511,10 @@ export function injectIntoDevTools(devToolsConfig: DevToolsConfig): boolean {
   const {ReactCurrentDispatcher} = ReactSharedInternals;
 
   return injectInternals({
-    ...devToolsConfig,
+    bundleType: devToolsConfig.bundleType,
+    version: devToolsConfig.version,
+    rendererPackageName: devToolsConfig.rendererPackageName,
+    findFiberByHostInstance: devToolsConfig.findFiberByHostInstance,
```

Comment:
This would be replaced with `rendererConfig` if we do https://github.com/facebook/react/pull/18233/files#r391135037